pythonAPIClient/client.py

```diff
@@ -115,33 +135,157 @@ def register(self, owner, permissionLevel='user'):
             Permission level of the owner of the client application.
             Default permission level is 'user'.
 
-        Raises
-        ------
 
         Returns
         -------
         str
             Client id of the newly registered client application.
         str
             Client secret of the newly registered client application.
+
         """
-        pass
+        url = Client.base_URL + 'apps/register'
```

Comment:
check if token exists in the beginning. or else the function will forever return with error saying 'no token provided'. same for other methods